docs/README.md

```diff
@@ -1,58 +1,44 @@
-# VTEX React App Template
```

Comment:
Before the app's name, you should add the following sentence: 

📢 Use this project, [contribute](https://github.com/vtex-apps/breadcrumb) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).